api/src/main/java/org/openmrs/module/registrationcore/api/RegistrationCoreService.java

```diff
@@ -140,13 +161,27 @@
 	 */
 	String importMpiPatient(String personId);
 
-    /**
-     * @return the engine used for biometric operations, if one is enabled
-     */
+	/**
+	 * Query to MPI server to find patient with Id "patientIdentifier" and specific patient identifier type
+	 * and save that patient to local DB.
+	 *
+	 * @param patientIdentifier person identifier of patient which should be imported
+	 * @param patientIdentifierTypeUuid identifier type uuid of given patientId
+	 * @return uuid of imported patient
+	 */
+	String importMpiPatient(String patientIdentifier, String patientIdentifierTypeUuid);
```

Comment:
I would expect this to return the Patient object that was imported and not just a uuid

Reply:
Thats true... I was just following the interface which was created originally by @approce during the GSOC project and that was not changed by the SolDevlo team when they were developing it further for iSantePlus. I have just changed it though, and changed the place which calls it in registration app as well.